source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb

```diff
@@ -16,7 +16,7 @@
       </td>
       <td class="doc-parameter-description">
         <p>The type of alternative fuel the station provides. Fuel types are given as code values as described below:</p>
-        <table border="0" cellpadding="0" cellspacing="0" class="doc-parameter-options">
+        <table border="0" cellpadding="0" cellspacing="0" class="table table-bordered">
```

Comment:
Thanks for finding these table style inconsistencies. If we're making these consistent, though, I think we should also make the other sub-tables up top in the requests area consistent too: https://developer.nrel.gov/docs/transportation/alt-fuel-stations-v1/all/#request-parameters https://github.com/NREL/developer.nrel.gov/blob/new-fields/source/docs/transportation/alt-fuel-stations-v1/_request_params_common2.html.md.erb#L17

Reply:
Updated table styles in request params